src/perception/include/cone_validator/z_score_validator.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <cone_validator/cone_validator.hpp>
+#include <utils/cluster.hpp>
+
+/**
+ * @brief The ZScoreValidator class is responsible for validating cones based on
```

Comment:
Maybe complete this a bit more because I was pretty confused about the goal of this validator

Reply:
OMW